lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -555,7 +555,7 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
 
     partition_service = LmsPartitionService(user=None, course_id=course_id)
     partitions = partition_service.course_partitions
-    group_configs_header = ['Group Configuration Group Name ({})'.format(partition.name) for partition in partitions]
+    group_configs_header = ['Experiment Group' for _ in partitions]
```

Comment:
I thought we were adding the partition name to the headers: https://openedx.atlassian.net/browse/TNL-1244?focusedCommentId=98645&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-98645